### PR TITLE
orm models defined

### LINKED ISSUE #14: Define ORM models
In the `blob_server` package, create a new Python module called `models.py`. Here is where we will define the ORM classes that will describe our database.

For this issue, you'll want to define classes to implement the data model described in #9. To see some introductory information on sqlalchemy ORM classes, check out the [Declare a Mapping](http://docs.sqlalchemy.org/en/latest/orm/tutorial.html#declare-a-mapping) section of sqlalchemy's ORM tutorial. For a real world example, see [my blog's data models](https://github.com/mplanchard/mplanchard/blob/master/mplanchard_server/models.py) (you can ignore all the schema stuff in there for now).

- [x] create a class for the `blog_posts` table with appropriate column attributes
- [x] create a class for the `tags` table with appropriate column attributes
- [x] create a many:many table linking the two (see [Basic Relationship Patterns](http://docs.sqlalchemy.org/en/latest/orm/basic_relationships.html#basic-relationship-patterns) at the sqlalchemy docs)

### PR DESCRIPTION
**Pull request by whichking**
resolves #14

### CODE REVIEW

File: blob_server/models.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+class BlogPost(Base):
+    """An entry containing the created date and the content of text"""
+    __tablename__ = 'blog_posts'
+
+    id = Column(Integer, primary_key=True)
+    created = Column(DateTime(timezone=True), server_default=func.now())
+    content = Column(Text)
+    tags = relationship(
+        'Tag',
+        back_populates='blog_posts',
```
Comment (mplanchard) [inline blob_server/models.py]: Your `back_populates` has to match the attribute name in the `Tag` class, in this case it should just be `posts`

File: blob_server/models.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+from sqlalchemy.orm import relationship
+
+base = declarative_base()
+
+
+PostTagLinks = Table(
+    'post_tag_links',
+    Base.metadata,
+    Column('post_id', Integer, ForeignKey(blog_posts.id)),
+    Column('tag_id', Integer, ForeignKey(tags.id))
```
Comment (mplanchard) [inline blob_server/models.py]: The `ForeignKey` argument should be a string

**Review by mplanchard — CHANGES_REQUESTED** [review]
Smoll changes required

**Comment by whichking** [comment]
Changes made

**Review by mplanchard — APPROVED** [review]
💾

**Pull request merged**